MediaEngineRMF/MediaEngineRMF.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
MediaEngineRMF.json should define the API spec in json similar to other plugins. For instance https://github.com/rdkcentral/rdkservices/blob/sprint/22Q4/PersistentStore/PersistentStore.json

The current contents of this file should be moved to MediaEngineRMFPlugin.json

Reply:
@anand-ky Will fix the JSON file. Thanks for the pointers.

Reply:
@anand-ky Updated JSON API spec and renamed/moved files as you suggested. Can you please check?